source-code/core/app/src/createInlang.ts

```diff
@@ -136,6 +137,23 @@ export const createInlang = async (args: {
 
 		const query = createReactiveQuery(() => messages()!)
 
+		let debouncedSave = (_newMessages) => {
+			// Skip initial call for setup
+			debouncedSave = debounce(500, async (newMessages) => {
+				// console.log('saving changes to messages')
+				try {
+					await resolvedModules()?.data.plugins.data.saveMessages({ messages: newMessages})
+				} catch (err) {
+					console.error("Error in saving messages ", err)
```

Comment:
Should we add this to the list of errors an app can show?
Please also use a dedicated class for the error.

Reply:
Oh yes, @janfjohannes add it to the error array. We should ban console.* logs in @inlang/app.